doc/compiler/optimizer/LocalOptimizationsSummary.md

```diff
@@ -0,0 +1,103 @@
+<!--
+Copyright (c) 2022, 2022 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Local Optimizations
+
+Local optimizations operate on extended basic blocks at a time.
+
+## Tree Simplification
+Tree Simplification performs simple IL tree transformations like constant
+folding, strength reduction, canonicalizing expressions (e.g. an `add` with a
+constant child will have the constant as the second child), branch/switch
+simplification etc. Float/double simplification support needs to be improved
+(call C libraries to compute the correct answer as per the Java spec).
```

Comment:
When you say support needs to be improved, are you calling out a shortcoming of the current state? If so, it probably doesn't make sense to have it in a documentation setting, unless you can reword it to describe what the current simplification does and that it has opportunity for further refinement.